config/README.md

```diff
@@ -381,6 +381,17 @@ test_extras = """
 """
 ```
 
+If your package uses [`mxdev`](https://pypi.org/project/mxdev/) to handle source checkouts for dependencies, you can set the `use_mxdev` key to ensure `tox` will first run mxdev.
+
+You also will need to manually set the installation of additional packages being pulled by `mxdev` in the `test_deps_additional` key.
+
+```toml
+[tox]
+use_mxdev = true
```

Comment:
If `use_mxdev` turns on `tox -e init` and that one calls always `mxdev -c mx.ini`, can be actually not ask for that setting to be manually set, but rather inspect the package and turn it on automatically (with a message printed about it) if `mx.ini` file is there?

IMHO, while trying to not be too clever, if sane _auto-magic defaults_ 🪄  can be obtained without too much user input can be done, all the best ✨ 🍀

Reply:
I would leave that for a second moment -- as someone could decide to not use mxdev even if the mx.ini is available  (i.e. `plone.distribution`)

Reply:
fair point 👍🏾